styleguide/core/Input.hbs

```diff
@@ -22,8 +22,8 @@
         <div class="{{blockName}}"{{extraAttributes}}>
 
             {{#defineBlockBody}}
-                {{element "input"}}
                 {{element "label"}}
+                {{element "input"}}
```

Comment:
shouldn't you style that based on the parent really? As in, you should write `.whateverParent .Input-label ` styles vs trying to do something more global with other selector types?